api/migrations/dev-data/data.sql

```diff
@@ -383,31 +383,31 @@ INSERT INTO public.content_node (id, slot, "position", instancename, createtime,
-	('ea0f65060295', '1', 0, NULL, '2022-02-04 19:49:02', '2022-02-04 19:49:02', '656295313d5c', '81dda09111d5', 'cfccaecd4bad', 'storyboard', '{"sections": {}}'),
-	('d1bc88c0f52e', '1', 0, NULL, '2022-02-04 19:49:34', '2022-02-04 19:49:34', '5bdd84d2ab79', '67be10a1b564', 'cfccaecd4bad', 'storyboard', '{"sections": {}}'),
-	('e37eb1a06e03', '1', 0, NULL, '2022-02-04 19:50:07', '2022-02-04 19:50:07', 'a29cff2d1d03', 'af6d5af59785', 'cfccaecd4bad', 'storyboard', '{"sections": {}}'),
-	('d245021c46fa', '1', 0, NULL, '2022-02-04 19:51:36', '2022-02-04 19:51:36', '6127f9640b45', '2d0de0759cd4', 'cfccaecd4bad', 'storyboard', '{"sections": {}}'),
-	('90bee6cb8815', '1', 0, NULL, '2022-02-04 19:51:56', '2022-02-04 19:51:56', 'e2ba5d9578b7', 'e44cf76f839d', 'cfccaecd4bad', 'storyboard', '{"sections": {}}'),
-	('0b625c957d44', '1', 0, NULL, '2022-02-04 19:52:59', '2022-02-04 19:52:59', '50172d027864', 'a7c05f91afbc', 'cfccaecd4bad', 'storyboard', '{"sections": {}}'),
-	('776dcb8f8664', '1', 0, NULL, '2022-02-04 19:53:09', '2022-02-04 19:53:09', '78904734f1c0', '6e929eaf43e5', 'cfccaecd4bad', 'storyboard', '{"sections": {}}'),
-	('1d40f6697b8c', '1', 0, NULL, '2022-02-04 19:53:15', '2022-02-04 19:53:15', '11d4984c77c6', 'f8e595ca2a2a', 'cfccaecd4bad', 'storyboard', '{"sections": {}}'),
-	('1dd5c52abd01', '1', 0, NULL, '2022-02-04 21:42:24', '2022-02-04 21:42:24', '8fa1c8e597c3', '0a23aa100e1b', 'cfccaecd4bad', 'storyboard', '{"sections": {}}'),
-	('9dda4a6262eb', '1', 0, NULL, '2022-02-04 21:43:13', '2022-02-04 21:43:13', 'fb21b1c912fd', '5598534f6d7f', 'cfccaecd4bad', 'storyboard', '{"sections": {}}'),
-	('a1488a153b81', '1', 0, NULL, '2022-02-04 21:59:10', '2022-02-04 21:59:10', '51d2e8dd6747', '48393c2e4cd6', 'cfccaecd4bad', 'storyboard', '{"sections": {}}'),
-	('8d07e57ba2f9', '1', 0, NULL, '2022-02-04 21:59:10', '2022-02-04 21:59:10', 'd0d0bef87638', 'cdc9597bd2c4', 'cfccaecd4bad', 'storyboard', '{"sections": {}}'),
-	('3160a58d2234', '1', 0, NULL, '2022-02-04 22:03:09', '2022-02-04 22:03:09', 'f9c28b8f6f31', '58a49366f78d', 'cfccaecd4bad', 'storyboard', '{"sections": {}}'),
-	('f3776b7ba11e', '1', 0, NULL, '2022-02-04 22:10:06', '2022-02-04 22:10:06', '37616c74811f', '4d88b71250c5', 'cfccaecd4bad', 'storyboard', '{"sections": {}}'),
-	('799121cc9c4b', '1', 0, NULL, '2022-02-04 22:28:12', '2022-02-04 22:28:12', '980885505aad', '82b8c7554597', 'cfccaecd4bad', 'storyboard', '{"sections": {}}'),
-	('0a86ef1c4503', '1', 0, NULL, '2022-02-04 23:12:11', '2022-02-04 23:12:11', '9b0e779f3238', '3790643b388a', 'cfccaecd4bad', 'storyboard', '{"sections": {}}'),
+	('e9b1e194ed58', '1', 0, NULL, '2022-02-04 19:11:47', '2022-02-04 19:11:47', '869c7bbcf042', '60434c561273', 'cfccaecd4bad', 'storyboard', '{"sections": {"074b541b-fc56-479f-b0ed-ae32b093927b": {"column1": "", "column3": "", "position": 1, "column2Html": ""}}}'),
```

Comment:
Like this the developers must delete their database an reinitialise it.
(and you had to fix it manually on dev).
If you increase the version of the migration filling in the dump, you don't have to do that, because the migration deletes the data and refills the seed data.

I have to document this somehow